t/property.t

```diff
@@ -0,0 +1,35 @@
+use t::WASM 'no_plan';
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: load
+--- config
+location /t {
+    content_by_lua_block {
+        local wasm = require("resty.proxy-wasm")
+        assert(wasm.load("plugin", "t/testdata/property/main.go.wasm"))
+    }
+}
+
+
+
+=== TEST2: get_property
+--- config
+location /t {
+    content_by_lua_block {
+        local wasm = require("resty.proxy-wasm")
+        local plugin = wasm.load("plugin", "t/testdata/property/main.go.wasm")
+        local plugin_ctx, err = wasm.on_configure(plugin, "test")
```

Comment:
Let's add a case which var is not found

Reply:
I added the missing callback when the variable does not exist, and also added test cases for it. Now, when the user requests a non-existent variable, the proxy-wasm-sdk will return an error.